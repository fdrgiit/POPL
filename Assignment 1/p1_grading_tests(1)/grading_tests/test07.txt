

WHILE.11REAL

>>>>>>>

0101010100INT.INT.REAL.IF.2INT

<<<<<<

