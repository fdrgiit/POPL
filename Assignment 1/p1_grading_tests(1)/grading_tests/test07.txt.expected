3 36 WHILE.11REAL 
5 32 >>
5 32 >>
5 32 >>
5 27 >
7 34 0 
7 34 101010100 
7 10 INT
7 33 .
7 10 INT
7 33 .
7 11 REAL
7 33 .
7 35 IF.2INT 
9 31 <<
9 31 <<
9 31 <<
